README.md

```diff
@@ -173,6 +175,70 @@ class { 'sensu':
 }
 ```
 
+### Installing Plugins
+
+Plugin management is handled by the `sensu::plugins` class.
+
+Example installing plugins on agent:
+
+```puppet
```

Comment:
should show how to disable the dependencies as many folks will want to do this because the packages can conflict

Reply:
Add docs to plugins section about passing `dependencies => []`.